### PR TITLE
Auto v2v case13018

### PR DESCRIPTION
**Pull request by mxie91**
Signed-off-by: mxie91 <mxie@redhat.com>

# Format of PR title < sub-system: summary >
e.g
*virsh_migrate: Fix unsupported direct socket mode issue*

# Check lists by category
## If the PR is new cases
- [ ] Description of the cases
- [ ] Links of libvirt features, libvirt bugs or case IDs
- [ ] Test results

## If the PR is bug cases
- [ ] Bug descriptions or bug links
- [ ] Test results

## If the PR is a trivial fix
It it is the fix of typos, comments or documents, the description of PR could be skipped.

### CODE REVIEW

**Comment by mxie91** [comment]
# avocado run --vt-type v2v function_test_esx.positive_test.iscsi_storage.esx_70.rhev.rhv_upload.it_vddk.vpx_uri
JOB ID     : b4254f8a81d90063a18e63a47d534df4e34cf496
JOB LOG    : /root/avocado/job-results/job-2021-07-19T13.34-b4254f8/job.log
 (1/1) type_specific.io-github-autotest-libvirt.function_test_esx.positive_test.iscsi_storage.esx_70.rhev.rhv_upload.it_vddk.vpx_uri:PASS (984.26 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 985.28 s

File: v2v/tests/cfg/function_test_esx.cfg [hunk 1]
```diff
@@ -233,6 +233,11 @@
                     version_requried = "[libguestfs-1.40,)"
                     only source_esx.esx_67
                     main_vm = VM_NAME_ESX_UEFI_RAID_V2V_EXAMPLE
+        - iscsi_storage:
+            only esx_70
+            boottype = 3
+            version_requried = "[libguestfs-1.40,)"
```
Comment (xiaodwan) [inline v2v/tests/cfg/function_test_esx.cfg:239]: This is a quite old case, so version_requried is not necessary.

**Comment by xiaodwan** [comment]
The PR summary is not a good title, Please update it to clearly reflect what the case is about.
And Put the case ID to the body part if you like.

File: v2v/tests/cfg/function_test_esx.cfg (hunk 1 above)
Comment (mxie91) [reply v2v/tests/cfg/function_test_esx.cfg:239]: Do I need to delete all codes about version_requried  in script?

Comment (xiaodwan) [reply v2v/tests/cfg/function_test_esx.cfg:239]: Yes, please.

File: v2v/tests/cfg/function_test_esx.cfg [hunk 2]
```diff
@@ -216,7 +216,6 @@
             removed_file = ESX_REMOVED_FILE_V2V_EXAMPLE
         - uefi:
             boottype = 3
-            version_requried = "[libguestfs-1.40,)"
```
Comment (xiaodwan) [inline v2v/tests/cfg/function_test_esx.cfg:219]: Please don't update the existing cases, only update the new case is fine.

**Comment by xiaodwan** [comment]
@mxie91
'case13018' is actually an internal info and not appropriate to be written into a title publicly.
You can add it into the body part.

**Review by xiaodwan — APPROVED** [review]

**Pull request merged**